spec/menu_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'menu'
+
+describe Menu do
+  subject(:menu) {described_class.new}
+  let(:default_menu) { {ribs: 3,
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.